aiida-backup.sh

```diff
@@ -4,15 +4,20 @@
 
 show_help() {
     echo
-    echo "Usage: ./aiida-backup.sh [-h] [-e env-type] [-c conda-env] [-v venv-path] [-p projects-path] [-n project-name]"
+    echo "Usage: ./aiida-backup.sh [-h] [-e env-type] [-c conda-env] [-v venv-path] [-p projects-name] [-n profile-names]"
```

Comment:
Good catch, though it should now be
```
echo "Usage: ./aiida-backup.sh [-h] [-e env-type] [-c conda-env] [-v venv-path] [-n project] [-p profiles]"
```

Reply:
Sure also okay. It is a bit sad that it is confusing together with `aiida-project`, but both suggestions have this confusion.